### PR TITLE
#2413 document the gradle commands used to build/test backend code only

### PR DESCRIPTION
**Pull request by chia7712**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2413)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: docs/src/how_to_build.rst [hunk 1]
```diff
@@ -70,6 +70,23 @@ to simplify the development of ohara.
    you testing your code on `official QA`_ which offers the powerful
    machine and IT envs.
 
+**Run All UTs for backend only**
```
Comment (vitojeng) [inline docs/src/how_to_build.rst]: Now we can use **Definition List** if here don't need a header.
https://ohara.readthedocs.io/en/latest/contrib-docs/quick_example.html#definition-lists

By the way, could you also check in this file?

File: docs/src/how_to_build.rst [hunk 2]
```diff
@@ -70,6 +70,23 @@ to simplify the development of ohara.
...
+
+.. code:: sh
```
Comment (vitojeng) [inline docs/src/how_to_build.rst]: Here we can use **console block**:
https://ohara.readthedocs.io/en/latest/contrib-docs/quick_example.html#console-block

File: docs/src/how_to_build.rst [hunk 3]
```diff
@@ -70,6 +70,23 @@ to simplify the development of ohara.
...
+
+   gradle clean test -PskipManager
```
Comment (vitojeng) [inline docs/src/how_to_build.rst]: After we use **console** code-blolck, we can add `$` now:
> **$** gradle clean test -PskipManager

File: docs/src/how_to_build.rst (hunk 1 above)
Comment (chia7712) [reply docs/src/how_to_build.rst]: There are some notes for different command, and I'm not sure whether the form "definition list" is able to accept "note" block. Hence, I keep the current style.

Comment (vitojeng) [reply docs/src/how_to_build.rst]: > I'm not sure whether the form "definition list" is able to accept "note" block.

Yes. Definition list accept "note" blcok.
You can reference the **docs/src/contrib-docs/quick_example.rst**. The whole quick example use definition list with many **block**, include code-block and note.

You also can build docs in the local for further checking.

File: docs/src/how_to_build.rst (hunk 3 above)
Comment (konekoya) [reply docs/src/how_to_build.rst]: Does this mean we have to add the `$` whenever we use the code-block?

**Pull request merged**